publish.js

```diff
@@ -473,6 +473,21 @@ var copyRecursiveSync = function(src, dest) {
     }
 };
 
+function getVersions () {
+    var topLevelOutDir = path.normalize(env.opts.destination);
```

Comment:
Thought about it.
I would love to have both options to get versions:

1. Getting versions from reading directories from the file system
2. Getting versions from the JSDoc-configuration file 

Cause I don’t think @SBoudrias wanted to add new versions to the JSDoc-configuration file with every release of [yeoman/generator](https://github.com/yeoman/generator) 😘